servers/quarkus-cli/src/main/resources/application.properties

```diff
@@ -20,9 +20,15 @@ nessie.server.default-branch=main
 nessie.server.send-stacktrace-to-client=false
 
 
-### which type of version store to use: INMEMORY, ROCKS, DYNAMO, MONGO.
+### which type of version store to use: INMEMORY, ROCKS, DYNAMO, MONGO, TRANSACTIONAL.
 nessie.version.store.type=INMEMORY
 
+## Transactional database configuration
+quarkus.datasource.db-kind=postgresql
```

Comment:
The `postgresql` setting is intentional, because only the `io.quarkus:quarkus-jdbc-postgresql` module is included. For another database type, if demanded, we (or they) would have to add other Quarkus dependencies/drivers and update the configuration (those are runtime settings).